### PR TITLE
Small fixes in `doc/themerc` and `docs/labwc-theme.5.scd`

### PR DESCRIPTION
**Pull request by tokyo4j**
- The default value of `osd.bg.color` is `#e1dedb`, as it falls back to `window.active.title.bg.color` (I may be missing some context here)
- "all border colors" in the description of `border.color` sounds like it includes `osd.border.color`

### CODE REVIEW

File: docs/labwc-theme.5.scd [hunk 1]
```diff
@@ -196,8 +196,9 @@ elements are not listed here, but are supported.
 	boxes. Default is 20.
 
 *border.color*
-	Set all border colors. This is obsolete, but supported for backward
-	compatibility as some themes still contain it.
+	Set both *window.active.border.color* and *window.active.border.color*.
```
Comment (Consolatis) [inline docs/labwc-theme.5.scd:199]: I assume one of them should read `window.inactive.border.color`.

Comment (tokyo4j) [reply docs/labwc-theme.5.scd]: Fixed.

**Comment by tokyo4j** [comment]
I added missing documentations on fallback values of `osd.bg.color`, `osd.border.color` and `osd.label.text.color`.

**Comment by johanmalm** [comment]
Thanks. Great tidy-ups. I think some of those go right back to the very, very early days of `labwc`.

It's possible that at the "early" time `border.color` did set "all" borders (that existed at the time) :smile:

And I possibly wasn't so careful with documentation either!

**Pull request merged**